pom.xml

```diff
@@ -149,8 +147,14 @@
   <reporting>
     <plugins>
       <plugin>
+        <!-- disable as it is not a Maven plugin project -->
         <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-invoker-plugin</artifactId>
+        <artifactId>maven-plugin-report-plugin</artifactId>
```

Comment:
In theory, it will be skippied anyway. Did you try?

Reply:
Yes I tried - it is skipped

Reply:
Without it - it is skipped but additional forked execution is run before skip.

Reply:
There is also now https://maven.apache.org/plugin-tools/maven-plugin-report-plugin/report-no-fork-mojo.html

Reply:
with no-fork report we can try with next parent

Reply:
Agreed